### PR TITLE
Give the Tk toolbar buttons a flat look

### PR DESCRIPTION
**Pull request by daniilS**
## PR Summary
This changes the parameters of the buttons in the Tk navigation toolbar to give them a flat look (with an outline showing when the mouse is over the button), similar to the Qt backend. For comparison:
Old:
old
New:
new

### CODE REVIEW

**Review by timhoffm — APPROVED** [review]
I think this looks nicer and more modern.

File: lib/matplotlib/backends/_backend_tk.py [hunk 1]
```diff
@@ -658,7 +658,10 @@ def _set_image_for_button(self, button):
 
     def _Button(self, text, image_file, toggle, command):
         if not toggle:
-            b = tk.Button(master=self, text=text, command=command)
+            b = tk.Button(
+                master=self, text=text, command=command,
+                relief="flat", overrelief="groove", borderwidth=1,
```
Comment (QuLogic) [inline lib/matplotlib/backends/_backend_tk.py:663]: Is the border width inside or outside the button? Just wondering if the size we allocate is consistent on HiDPI screens?

Comment (daniilS) [reply lib/matplotlib/backends/_backend_tk.py:663]: It's outside of the button, and the default width seems to be 2, so space shouldn't be an issue.

Comment (QuLogic) [reply lib/matplotlib/backends/_backend_tk.py:663]: I'm more concerned about the height of the toolbar than the width of the border itself.

Comment (daniilS) [reply lib/matplotlib/backends/_backend_tk.py:663]: The toolbar height is set to the height of two text lines [here](https://github.com/matplotlib/matplotlib/blob/5555daa96aa9b4b57a652147d1dab53798272c1f/lib/matplotlib/backends/_backend_tk.py#L551), which on all platforms I've tried is bigger than the size of the buttons, so the buttons becoming 2px smaller shouldn't affect the height of the toolbar.

**Review by QuLogic — APPROVED** [review]

**Pull request merged**